app/routes.py

```diff
@@ -1,2 +1,218 @@
+    customer = Customer.query.get_or_404(customer_id)
+
+
+    if not customer:
+        return jsonify({
+            "Customer info invalid. Try again!"
+        }, 404)
+
+    if request.method == "GET":
+        return jsonify(
+            customer.resp_json()
+        ), 200
+    
+    elif request.method == "PUT":
+        request_body = request.get_json()
+        if "name" not in request_body or "postal_code" not in request_body or "phone" not in request_body:
+            return make_response({
+                "details": "invalid data"
+            }, 400)
+
+        customer.name = request_body["name"]
+        customer.postal_code = request_body["postal_code"]
+        customer.phone = request_body["phone"]
+        db.session.commit()
+
+        return jsonify(
+            customer.resp_json()
+        ), 200
+
+    elif request.method == "DELETE":
+        db.session.delete(customer)
+        db.session.commit()
+        
+        return jsonify({
+            "id": customer.customer_id
+        }), 200
+
+
+                                              ### CRUD VIDEO ###
+
+@videos_bp.route("", methods=["GET", "POST"], strict_slashes = False)
+def video_main_func():
+
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body or "release_date" not in request_body or "total_inventory" not in request_body:
+            return make_response({
+                "details": "invalid data"
+            }, 400)
+
+        single_video = Video(
+        title= request_body["title"],
+        release_date = request_body["release_date"],
+        total_inventory = request_body["total_inventory"]
+        )
+        db.session.add(single_video)
+        db.session.commit()
+        response = {
+            "id": single_video.video_id
+        }
+        return jsonify(response), 201
+
+    elif request.method == "GET":
+        all_videos =Video.query.all()
+        response = []
+
+        for video in all_videos:
+            response.append(video.resp_json())
+        return jsonify(response), 200
+
+
+@videos_bp.route("/<video_id>", methods=["GET", "PUT", "DELETE"], strict_slashes = False)
+def video_func(video_id):
+    video = Video.query.get(video_id)
+    
+    if not video:
+        return make_response("", 404)
+
+    if request.method == "GET":
+        return jsonify(
+            video.resp_json()
+        ), 200
+
+    elif request.method == "PUT":
+        about_video = request.get_json()
+        if "title" not in about_video or "release_date" not in about_video or "total_inventory" not in about_video:
+            return make_response({"details": "invalid data"}, 400)
+
+        video.title = about_video["title"]
+        video.release_date = about_video["release_date"]
+        video.total_inventory = about_video["total_inventory"]
+        
+        db.session.commit()
+        return jsonify(
+            video.resp_json()
+        ), 200
```

Comment:
`video.resp_json()` returns a dictionary. Flask actually turns all dictionaries into json objects when used as response objects so technically you could omit `jsonify()`.
```suggestion
        return video.resp_json(), 200
```